source/includes/fact-view-query-conversion-history.rst

```diff
@@ -0,0 +1,19 @@
+You can view the history of previous conversions in the left-hand 
+:guilabel:`Query Converter` pane. Each conversion has an icon 
+indicating the result of the conversion. If an object does not have 
+an icon next to it, a conversion has not been attempted yet:
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Icon 
+     - Description
+
+   * - Spinner
+     - The conversion is currently being executed.
```

Comment:
[optional nit]
I like Vale's suggestion here. "Now" is a simpler word than "currently".